tests/lib/Persistence/Legacy/Notification/Gateway/DoctrineDatabaseTest.php

```diff
@@ -145,6 +146,72 @@ public function testLoadUserNotifications()
         ], $results);
     }
 
+    public function testFindUserNotifications(): void
+    {
+        $userId = 14;
+        $offset = 1;
+        $limit = 3;
+        $queryWithoutFilters = new NotificationQuery([], $offset, $limit);
+
+        $resultsWithoutQuery = $this->getGateway()->findUserNotifications($userId, $queryWithoutFilters);
+
+        $this->assertEquals([
```

Comment:
Static methods should be called statically (yes we have a lot of technical debt in that regard :D)
```suggestion
        self::assertEquals([
```